google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
do we need to update the year @chingor13?

Reply:
This should stay 2016 - it's the year the file was first created/copyrighted

Reply:
Thank you @kolea2 @chingor13. I will change it back to 2016.